### DISCUSSION TITLE
Emulation support and Jetpack 5.

### DISCUSSION THREAD

**Discussion by DarkCowMoo** [OE4T/meta-tegra#1025] [Q&A] [answered]
Hi,
My team has been using a custom Jetpack image (edited manually) for quite some time which is something I am supposed to change.
I would like to use Yocto to customize the Jetpack image and was wondering if it is possible to enable emulation using the meta-tegra layer. I was not able to to run runqemu after compiling the layer as there is no .emuconf file (not the exact file extension).
I was also wondering why I wasn't referred to this board which asking about customizing Jetpack in the official Nvidia forums. Is meta-tegra not fully supported? 
Thanks,
Shauli

**Comment by madisongh** [comment, marked as answer]
If you're asking about using QEMU to emulate a Jetson device, then no, there is no such emulation available.

As for meta-tegra being "fully supported" - this is a community project, maintained and run by users of the layer, so it's not directly sponsored by NVIDIA.  They do work with us, though, and some of the participants here are NVIDIA partner companies.

**Reply by DarkCowMoo** [reply]
Thanks for answering. The work you did is amazing. Hopefully, I will be able to contribute in future.
